tapdb/sqlc/migrations/000022_transfer_outputs_proof_delivered.up.sql

```diff
@@ -0,0 +1,28 @@
+-- Add a column to track if the proof has been delivered for an asset transfer
+-- output.
+ALTER TABLE asset_transfer_outputs
+ADD COLUMN proof_delivery_complete BOOL;
+
+-- Add column `position` which indicates the index of the output in the list of
+-- outputs for a given transfer. This index position in conjunction with the
+-- transfer id can be used to uniquely identify a transfer output.
+--
+-- We'll be inserting an actual value in the next query, so we just start
+-- with -1.
+ALTER TABLE asset_transfer_outputs
+ADD COLUMN position INTEGER NOT NULL DEFAULT -1;
```

Comment:
Why not use a `BIGINT` here? Which should translate to an `int64` on the Go side, and reduce the need for some of the type checking elsewhere in the PR.

Reply:
As a counterargument, I don't think we can reasonably perform a transfer with `2^32` outputs anyways.  I'm not sure if there's an argument for supporting that.